package.json

```diff
@@ -7,13 +7,16 @@
   "main": "core/core.bundle.js",
   "scripts": {
     "build:dev": "webpack -d --config webpack.dev.js",
-    "build:prod": "webpack -p --config webpack.prod.js",
-    "dist": "node build.js",
-    "clean": "rimraf apps/*/node_modules apps/*/dist dist/ node_modules",
-    "update": "node update.js",
+    "build": "webpack -p --config webpack.prod.js",
+    "dist": "run-s clean-all install-all build-all",
+    "clean-all": "rimraf apps/*/node_modules apps/*/dist dist/",
+    "update-all": "node build/run-for-all.js npm-check-updates -a",
+    "install-all": "node build/run-for-all.js yarn install",
+    "build-all": "node build/run-for-all.js yarn build",
+    "watch-all": "node build/run-for-all.js -p yarn watch",
     "watch": "yarn run build:dev && webpack-dev-server --progress --colors --watch --config webpack.dev.js --public $SERVER_HOST",
-    "lint": "eslint src/**/*.vue src/**/*.js src/*.vue src/*.js \"tests/**/*.js\" --color",
-    "lint-fix": "eslint src/**/*.vue src/**/*.js src/*.vue src/*.js \"tests/**/*.js\" --color --fix",
+    "lint": "eslint build/**/*.js src/**/*.vue src/**/*.js src/*.vue src/*.js \"tests/**/*.js\" --color",
```

Comment:
we maybe should change this line to 
`eslint build src tests --ext vue --ext js --color` and same to the other line with `--fix`

Reply:
pull request welcome ;-)